src/main/java/codeu/model/store/basic/ConversationStore.java

```diff
@@ -100,4 +102,15 @@ public Conversation getConversationWithTitle(String title) {
   public void setConversations(List<Conversation> conversations) {
     this.conversations = conversations;
   }
+  //list of conversations created by user
```

Comment:
Watch out for indenting 4 spaces instead of 2!

Reply:
done